crd2pulumi-crontabs/kubernetes-go-crontabs/main.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
+
+package main
+
+import (
+	crontabs "kubernetes-go-crontabs/crontabs/v1"
```

Comment:
```suggestion
	crontabsv1 "kubernetes-go-crontabs/crontabs/v1"
```

Multiple versions are common, so it's generally best to include that as part of the alias.